.editorconfig

```
@@ -22,7 +22,14 @@ indent_style = tab
 indent_size = 4
 max_line_length = 79
 
-[jobs/*]
+[dev/jobs/*]
+indent_style = space
+indent_size = 4
+shell_variant = bash
+switch_case_indent = true
+space_redirects = true
+
+[dev/job_cards/rocoto/*]
```

```suggestion
[dev/jobs/*]
```
As @DavidHuber-NOAA points out `job_cards/rocoto/*` are covered under `*.[sh,bash]`